### PR TITLE
Update area-owners.md

### LINKED ISSUE #26498: Update area-owners.md
Looks like this file is out of date: https://github.com/dotnet/aspnetcore/blob/master/docs/area-owners.md

@Pilchie can you add new owners?

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: docs/area-owners.md [hunk 1]
```diff
@@ -2,22 +2,22 @@ The below table lists all the `area-`labels used in the `aspnetcore` repository
...
 | area-infrastructure | dougbu | |
-| area-installers | anurse | |
-| area-middleware | anurse | |
+| area-installers | dougbu | |
+| area-middleware | BrennanConroy | |
 | area-mvc | mkArtakMSFT | |
-| area-perf | anurse | |
-| area-platform | anurse | |
+| area-perf | sebastienros | |
+| area-platform | BrennanConroy | |
```
Comment (BrennanConroy) [inline docs/area-owners.md:19]: This one is being monitored by @JunTaoLuo

File: docs/area-owners.md [hunk 2]
```diff
@@ -2,22 +2,22 @@ The below table lists all the `area-`labels used in the `aspnetcore` repository
 
 | Area label | Owner | Description|
 |--- | ---| --- |
-| area-azure | anurse | |
+| area-azure | BrennanConroy | |
 | area-blazor | mkArtakMSFT | Blazor server and Blazor WASM related |
-| area-commandlinetools | anurse, mkArtakMSFT | dev certs, dotnet watch,  |
-| area-dataprotection | anurse | |
+| area-commandlinetools |  mkArtakMSFT | dev certs, dotnet watch,  |
+| area-dataprotection | BrennanConroy | |
```
Comment (BrennanConroy) [inline docs/area-owners.md:8]: @blowdart Owns this one I think

File: docs/area-owners.md [hunk 3]
```diff
@@ -2,22 +2,22 @@ The below table lists all the `area-`labels used in the `aspnetcore` repository
...
 | area-grpc | shirhatti | |
-| area-healthchecks | rynowak | |
-| area-hosting | anurse | |
-| area-httpclientfactory | anurse | |
+| area-healthchecks | mkArtakMSFT | |
+| area-hosting | BrennanConroy | |
+| area-httpclientfactory | mkArtakMSFT | |
```
Comment (BrennanConroy) [inline docs/area-owners.md:12]: We've been looking at this one, also I think we should remove it. The only part we own is Polly integration which is in the Extensions repo, and Runtime has the rest.

File: docs/area-owners.md (hunk 2 above)
Comment (blowdart) [reply docs/area-owners.md:8]: If this is engineering ownership, no I don't. 

@Pilchie has yet to assign anyone, so I'd assign it to him :)

File: docs/area-owners.md (hunk 1 above)
Comment (JunTaoLuo) [reply docs/area-owners.md:19]: Yea this should be me.

File: docs/area-owners.md (hunk 2 above)
Comment (pranavkm) [reply docs/area-owners.md:8]:
```suggestion
| area-dataprotection | Pilchie | |
```

File: docs/area-owners.md (hunk 3 above)
Comment (pranavkm) [reply docs/area-owners.md:12]:
```suggestion
```

File: docs/area-owners.md (hunk 1 above)
Comment (pranavkm) [reply docs/area-owners.md:19]:
```suggestion
| area-platform | JunTaoLuo | |
```

**Review by BrennanConroy — APPROVED** [review]

**Pull request merged**